velox/exec/SortBuffer.cpp

```diff
@@ -133,7 +140,6 @@ void SortBuffer::noMoreInput() {
     // Finish spill, and we shouldn't get any rows from non-spilled partition as
     // there is only one hash partition for SortBuffer.
     Spiller::SpillRows nonSpilledRows = spiller_->finishSpill();
-    VELOX_CHECK_LE(spiller_->stats().spilledPartitions, 1);
```

Comment:
Why we can't have this check?

Reply:
Because right now OrderBy gets the stats and do the check as well, so this is redundant which is why I removed it